recipe/meta.yaml

```diff
@@ -58,17 +58,17 @@ requirements:
     - azure-identity-cpp
     - azure-storage-blobs-cpp
     - bitmagic
-    - krb5 =1.20.1
+    - krb5
```

Comment:
krb5 is a crucial package underlying lots of fundamental packages; if you pin it (for longer than absolutely necessary; assuming anything breaks), you doom yourself.

Reply:
ArcticDB does not use Kerberos directly. It's a transitive dependency of OpenSSL.

Would it work if we remove this explicit dependency?

Reply:
We do not need to include transitive dependency, I think this one was added to work around a bug. Let's try remove to remove it now.

Reply:
`cyrus-sasl =2.1.27` might also be removable as a pin.

Reply:
OK cool. There's a bunch more things that the link check points out as not-actually-linked to. Those too would be candidates for removal, which would also reduce the "surface" of migrations you're exposed to.

Dependencies have a cost (not just in conda-forge BTW), so you shouldn't declare them (and then pay the price for having them) if you don't need them. ;-)

Reply:
> Dependencies have a cost (not just in conda-forge BTW), so you shouldn't declare them (and then pay the price for having them) if you don't need them. ;-)

Well, you are preaching to the choir; but some decisions makers do not agree with this point due to other constraints.